source/apps/by_team.html.md.erb

```diff
@@ -0,0 +1,29 @@
+---
+layout: application_layout
+title: Application Ownership by Team
```

Comment:
I think the other pages use non-titleised headings, maybe `Application ownership by team` is more consistent?